From a76b7bbaa6051e8511bdf43f5fe31db131b5e424 Mon Sep 17 00:00:00 2001
From: duckyduckG <duckyduckg65@gmail.com>
Date: Fri, 10 Jan 2025 14:20:50 +0500
Subject: [PATCH 165/175] defconfig: Switch to LEDS QPNP HAPTICS drivers: leds:
 Import QPNP haptics driver from diphons repo LEDS: Implement userspace
 vibration control

Node: /sys/module/leds_qpnp_haptics/parameters/haptic_gain
 Default value : 80
Node: /sys/module/leds_qpnp_haptics/parameters/haptic_gain_show
---
 arch/arm64/configs/vendor/xiaomi/beryllium.config | 15 ++-------------
 1 file changed, 2 insertions(+), 13 deletions(-)

diff --git a/arch/arm64/configs/vendor/xiaomi/beryllium.config b/arch/arm64/configs/vendor/xiaomi/beryllium.config
index 6b58763aef13..73919762feb0 100644
--- a/arch/arm64/configs/vendor/xiaomi/beryllium.config
+++ b/arch/arm64/configs/vendor/xiaomi/beryllium.config
@@ -26,20 +26,9 @@ CONFIG_BACKLIGHT_QCOM_WLED=y
 CONFIG_INPUT_QTI_HAPTICS=n
 CONFIG_LEDS_QPNP_HAPTICS=y
 
-# Kprobes
-CONFIG_KPROBES=y
-CONFIG_HAVE_KPROBES=y
-CONFIG_KPROBE_EVENTS=y
-
-# KernelSU_next
-CONFIG_KSU=n
-
-# Susfs
-CONFIG_KSU_SUSFS=n
-CONFIG_KSU_SUSFS_TRY_UMOUNT=n
-CONFIG_KSU_SUSFS_HAS_MAGIC_MOUNT=n
-
 #
 # RCU Debugging
 #
 CONFIG_DISABLE_TRACE_PRINTK=n
+
+
-- 
2.48.1

